manifest.json

```diff
@@ -5,6 +5,17 @@
             "alias": "local",
             "type": "runtime",
             "programs": [
+                {
+                    "id": "install-deps",
+                    "executable": "python3",
+                    "args": [
```

Comment:
Should we better align if we want to use `python`or `python3`
(compare with https://github.com/eclipse-velocitas/devenv-devcontainer-setup/pull/89/files)

I guess it does not have any practical difference as I assume python and python3 both exists and points to the same executable in all of our environments, so it is more of a philosophical question

Reply:
Talked to Dominic: We should use `python3`

Reply:
Will adapt that in devenv-devcontainer-setup ...